src/gleam/date.gleam

```diff
@@ -0,0 +1,136 @@
+/// Date is type that is similar in functionality to the Erlang calendar module,
+/// but the types do not directly interface with the calendar module.
+/// To convert an erlang date to a Date type, use `from_tuple/1`
+///
+/// Currently this module defaults to UTC.
+///
+/// Years cannot be abreviated.  Years start from 0 and go up.  So if the year = 93,
+/// it does not refer to 1993.  This is in line with the behavior of Erlang.
+///
+import gleam/result
+import gleam/int
+
+pub type Date {
+  Date(year: Int, month: Int, day: Int)
+}
+
+/// Gets the UTC timestamp.
+///
+/// ## Example
+///
+/// ```gleam
+/// universal_time() = tuple(tuple(year, month, day), tuple(hour, minute, second))
+/// ```
+///
+external fn universal_time() 
```

Comment:
As I am working on the `Time` module, this will have to be moved to an external module, something like `datetime_utils`...